dizoo/atari/example/atari_dqn_dist_rdma.py

```diff
@@ -62,10 +66,15 @@ def main():
             task.use(eps_greedy_handler(cfg))
             task.use(StepCollector(cfg, policy.collect_mode, collector_env))
             task.use(termination_checker(max_env_step=int(1e7)))
+            time.sleep(10)  # give some time to learner recving
         else:
             raise KeyError("invalid router labels: {}".format(task.router.labels))
 
-        task.run()
+        __start_time = time.time()
```

Comment:
you can use `EasyTimer`, just `from ding.utils import EasyTimer`

Reply:
Done